_posts/2023-09-08-trino-summit-2023-announcement.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "Join us for Trino Summit 2023"
+author: "Manfred Moser"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-20232/summit-logo.png
+---
+
+The Trino community is buzzing. Commander Bun Bun is ready to invite you all to
+join us for Trino Summit 2023. And "all" really means everyone in the community.
+The event is free to attend, virtual, and full of news and shared knowledge from
+your peers using Trino. Don't hesitate, submit your talk, and register to attend
+now.
+
+<!--more-->
+
+We are pleased to announce the upcoming Trino Summit 2023. The summit is
+scheduled as virtual event on the 13th and 14th of December 2023, and attendance is
+free!
+
+If you'd like to share your knowledge and information about Trino usage and give
+a talk at this year's Trino Summit, we're putting out a call for speakers. We
+are accepting submissions from now until the 22nd of October, but we recommend
+submitting soon, because slots are filling up fast.
```

Comment:
```suggestion
submitting as soon as possible, because we expect slots to fill up fast.
```
Let's not lie and say they're filling up fast when we haven't even opened it up yet. 😆